_posts/2024/2024-02-06-about-caching-in-eo.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+date: 2024-02-06
+title: "Build cache in EO and other build systems"
+author: Alekseeva Yana
+---
+
+
+## Introduction 
+In [EO](https://github.com/objectionary/eo), caching is used to speed up program compilation.
+Recently we found a caching 
+[bug](https://github.com/objectionary/eo/issues/2790) in `eo-maven-plugin`
+for EO version `0.34.0`. The error occurred because the algorithm compared
+the compilation time and caching time to search for the needed file.
```

Comment:
@Yanich96 
This sentence is a bit strange:
> The error occurred because the algorithm compared
 the compilation time and caching time to search for the needed file.

Which algorithm?  Which file do you mean?